pkg/sshd/server_test.go

```diff
@@ -29,6 +27,27 @@ func TestGitPktLine(t *testing.T) {
 	assert.Equal(t, outStr[4:], str, "remainder of string")
 }
 
+func serverconfigure() (*ssh.ServerConfig, error) {
+	cfg := &ssh.ServerConfig{
+		PasswordCallback: func(c ssh.ConnMetadata, pass []byte) (*ssh.Permissions, error) {
+			// Should use constant-time compare (or better, salt+hash) in
+			// a production setting.
```

Comment:
This is for convenience we are doing. We can do public key call back but adds more code in the tests. SO password callback is easy All I wanted was to get ssh.permissions object which is returned only by callback functrions